### DISCUSSION TITLE
How do I find my Account ID

### DISCUSSION THREAD

**Discussion by simondean** [BottlecapDave/HomeAssistant-OctopusEnergy#602] [General]
Hi.  I'm not sure what to enter in the `Account ID` field when setting up the integration in Home Assistant: 

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/190464/4855645f-a06c-4c7f-94dc-d53fe123ce8e)

The https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/setup_account.md page links to https://octopus.energy/dashboard/new/accounts/personal-details/api-access which provides my `API Key` but I can't seem to find somewhere that shows my `Account ID`.  What should look to find my Octopus Account ID?  Thanks

**Comment by simondean** [comment]
I've just found my Octopus Account ID on my latest bill 😄.  Might be worth mentioning on https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/setup_account.md that the Account ID can be found on your Octopus bill?  Thanks!

**Reply by JRascagneres** [reply]
You just login online and it's at the top of the page

**Reply by BottlecapDave** [reply]
Sorry you had trouble finding this information. I've updated the config screen to have links to the appropriate sections to help future users.
